### ISSUE TITLE
SC cannot be evaluate to inapplicable ?

### ISSUE THREAD

**Issue by audreymaniez** [w3c/wcag-act#240]
In the 14.3 Outcome section : 

> _While inapplicable is a valid result for ACT Rules, it may not be a valid result for all accessibility requirements. Notably *the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to true (passed) or false (failed)*. This translation of results is part of output aggregation_

I'm sorry if i misunderstood, but I do have miss a point ... I don't understand why a SC can only be evaluated to passed or failed ?  Why *innaplicable* is not a valid outcome for a SC ? 

What about a website/page without video: SC like 1.2.1 cannot be set to passed or failed ?

**Comment by nitedog** [comment]
If a web page does not include applicable content, then it "satisfies" the corresponding WCAG 2 requirement. See: https://www.w3.org/TR/WCAG21/#dfn-satisfies

**Comment by audreymaniez** [comment]
Thank you ! So sorry that i missed that point ... too used to our french evaluation method that is a little bit different on that point.
